showcase/.eslintrc.js

```diff
@@ -26,10 +26,26 @@ module.exports = {
         functions: 'only-multiline',
       },
     ],
-    // ember doesn't support template only components that have type signatures so we need to use empty component classes
-    'ember/no-empty-glimmer-component-classes': 'off',
   },
   overrides: [
+    //  gts files
+    {
+      files: ['**/*.gts'],
```

Comment:
Note: I based the overrides on the HDS+ repo https://github.com/hashicorp/hds-plus/blob/5bbd63af89f581f52ed1983d777992920e3e7c60/showcase/.eslintrc.js#L42